File: pizzeria/pizzeria-core/src/main/java/io/github/joxit/pizzeria/dto/PizzaDTO.java
package io.github.joxit.pizzeria.dto;

import java.util.Collection;

/**
 * @author Jones Magloire @Joxit
 * @since 2017-11-01
 */
public class PizzaDTO {

  private String name;
  private float prix;
  private Collection<String> ingredients;

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public float getPrix() {
    return prix;
  }

  public void setPrix(float prix) {
    this.prix = prix;
  }

  public Collection<String> getIngredients() {
    return ingredients;
  }

  public void setIngredients(Collection<String> ingredients) {
    this.ingredients = ingredients;
  }

  @Override
  public String toString() {
    return "PizzaDTO{" +
        "name='" + name + '\'' +
        ", prix=" + prix +
        ", ingredients=" + ingredients +
        '}';
  }
}

File: pizzeria/pizzeria-core/src/main/kotlin/io/github/joxit/pizzeria/dto/PizzaDTO.kt
package io.github.joxit.pizzeria.dto

/**
 * Old Java version : https://github.com/Joxit/IG-Master2/blob/promo-2021/pizzeria/pizzeria-core/src/main/java/io/github/joxit/pizzeria/
 * @author Jones Magloire (@Joxit)
 * @since Java: 2017-11-01
 * @since Kotlin: 2022
 */
data class PizzaDTO(
  val name: String,
  val prix: Float,
  val ingredients: Collection<String>
)